cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1101,7 +1101,6 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
           binaryBuf.valueCapacity = capacity;
           dstValuePtr = binaryBuf.valuePtr + valueOffset - stringLen;
 
-          DLOG(INFO) << "Split value buffer resized. Column index: " << binaryColumnIndices_[binaryIdx] << std::endl;
           VsPrintSplit(" dst_start", dstOffsetBase[x]);
```

Comment:
Let's remove the vsprintsplit as well, it's just for debug purpose

Reply:
and with new method, it's expected less likely to resize the binary buffer

Reply:
Seems like this issue is introduced by the new method.